src/rgw/rgw_rados.cc

```diff
@@ -4806,6 +4806,10 @@ int RGWRados::open_bucket_index_ctx(const RGWBucketInfo& bucket_info, librados::
   }
   auto iter = zone_params.placement_pools.find(*rule);
   if (iter == zone_params.placement_pools.end()) {
+    rule = &zonegroup.default_placement;
```

Comment:
does this necessary?  The ` default_placement` of  `bucket_info`  should be setup correctly during bucket creation.